Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -38,6 +38,7 @@ struct SignalArray {
   double firstTBtime;                     // first TB time
   std::unordered_map<int, int> trackIds;  // tracks Ids associated to the signal
   std::vector<MCLabel> labels;            // labels associated to the signal
+  bool writtenOut{false};                 // whether this signal has been written out to an event.
```

Comment:
Please, remove this; all signals are written out, and the implementation of the pileup will take care of event triggering.

Reply:
oops that was me buggering around trying to fix labels.
removing.